Utilities/Pipelines/libHttpClient.CI.yml

```diff
@@ -0,0 +1,168 @@
+          Platform: x64
+          Configuration: Debug
+        x86_Release:
+          Platform: x86
+          Configuration: Release
+        x64_Release:
+          Platform: x64
+          Configuration: Release
+    steps:
+      - template: Tasks/vs2017-build.yml
+        parameters:
+          platform: $(Platform)
+          configuration: $(Configuration)
+
+  ####################
+  # Visual Studio 2019
+  ####################
+
+  - job: Win32Vs19Build
+    displayName: libHttpClient Win32/UWP VS2019 Build
+    pool:
+      name: xbl-1es-vs2019
+    timeoutInMinutes: 180
+    strategy:
+      matrix:
+        x86_Debug:
+          Platform: x86
+          Configuration: Debug
+        x64_Debug:
+          Platform: x64
+          Configuration: Debug
+        x86_Release:
+          Platform: x86
+          Configuration: Release
+        x64_Release:
+          Platform: x64
+          Configuration: Release
+    steps:
+      - template: Tasks/vs2019-build.yml
+        parameters:
+          platform: $(Platform)
+          configuration: $(Configuration)
+
+  ####################
+  # Android
+  ####################
+
+  - job: AndroidBuild
+    displayName: libHttpClient Android Build
+    pool:
+      vmImage: internal-macos-10.15
+    timeoutInMinutes: 180
+    strategy:
+      matrix:
+        x64_Release:
+          Platform: x64
+          Configuration: Release
+        x64_Debug:
+          Platform: x64
+          Configuration: Debug
+    steps:
+      - template: Tasks/android-build.yml
+        parameters:
+          platform: $(Platform)
+          configuration: $(Configuration)
+    
+  ####################
+  # iOS
+  ####################
+
+  - job: iOSBuild
+    displayName: libHttpClient iOS Build
+    pool:
+      vmImage: internal-macos-10.15
+    timeoutInMinutes: 180
+    strategy:
+      matrix:
+        Debug:
+          Configuration: Debug
+    steps:
+      - template: Tasks/ios-build.yml
+        parameters:
+          configuration: $(Configuration)
+    
+  ####################
+  # XDK
+  ####################
+
+  - job: XDKBuild
+    displayName: libHttpClient XDK Build    
+    pool:
+      name: xbl-1es-vs2017-xdk
+    timeoutInMinutes: 180
+    strategy:
+      matrix:
+        Durango_Release:
+          Platform: Durango
+          Configuration: Release
+        Durango_Debug:
+          Platform: Durango
+          Configuration: Debug
+    steps:
+      - template: Tasks/xdk-build.yml
+        parameters:
+          platform: $(Platform)
+          configuration: $(Configuration)
+
+  #########################
+  # Send email on failure
+  #########################
+
+  - job: SendEmailOnFailure
+    displayName: Send email on build failure
+    condition: and(failed(), eq(variables['Build.SourceBranch'], 'refs/heads/main'))
+    dependsOn:
+      - Win32Vs17Build
+      - Win32Vs19Build
+      - AndroidBuild
+      - iOSBuild
+      - XDKBuild
+    pool:
+      name: xbl-1es-vs2019
```

Comment:
Will this work on the `1es` machines? Wondering since the email-sending is a PackageES task, I thought this had to run on a PackageES machine

Reply:
Yeah works fine.  PackageES has made their tasks work on 1ES as part of migration to 1ES